torch/_torch_docs.py

```diff
@@ -4029,6 +4029,7 @@ def merge_dicts(*dicts):
 
     Unlike NumPy's dot, torch.dot intentionally only supports computing the dot product
     of two 1D tensors with the same number of elements.
+    Keyword argument of torch.dot is unsupported.
```

Comment:
I don't think they're unsupported, updating the signature above to say input/tensor instead of input/other would work no? (even though the name is unfortunate of course).

Reply:
How about "User should pass tensor instead of use python keyword argument", does this look good? :D

Reply:
Why not just update the signature above line 4024 to match what you can actually use?

Reply:
@albanD Like this one?
```
line 4024   dot(input, tensor, *, out=None) -> Tensor
```

I see some other functions in doc use input/other pair as params name, is this new pair(input/tensor) may case inconsistent or confusion for users? 

If we don't need to worry about that, I will update my change.

Thanks for your time! :D

Reply:
> I see some other functions in doc use input/other pair as params name, is this new pair(input/tensor) may case inconsistent or confusion for users?

It will indeed be inconsistent and confusing.
But I don't think it is worth the BC-breaking change to change how the user can call this function.

Reply:
@albanD Changes applied, does CI able to regenerate preview doc, maybe it's better to see the real content instead of code before merge it, thanks!

Reply:
Yes, you'll be able to see the preview from https://github.com/pytorch/pytorch/pull/125908#issuecomment-2104159407 once the build is done

Reply:
Thanks, checked It works as expect. Maybe it can be merged if everything is ok 👯